# each scan point

uint8   layer
uint8   echo
uint8   flags
uint16  horizontal_angle
uint16  radial_distance
uint16  echo_pulse_width
